flexmailer.php

```diff
@@ -190,6 +190,15 @@ function flexmailer_civicrm_navigationMenu(&$menu) {
   _flexmailer_civix_navigationMenu($menu);
 } // */
 
+/**
+ * Implements hook_civicrm_alterMenu().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_alterMenu
+ */
+function flexmailer_civicrm_alterMenu(&$items) {
+  $items['civicrm/mailing/view']['page_callback'] = '\Civi\FlexMailer\FlexMailer::viewMailing';
```

Comment:
I haven't really tested (`r-run`) yet, but regarding code-style (`r-code`)... could this function go in a new class? Maybe `\Civi\FlexMailer\Page\ViewMailing` (`src/Page/ViewMailing.php`)?

I think of the `FlexMailer.php` as akin to core's `Civi\API\Kernel` or Symfony's `HttpKernel` -- its main job is to emit events for the delivery algorithm,  but most substantive work should be handled by separate classes.

Reply:
Updated the PR with the above suggestion. Thanks 👍